argon2/src/lib.rs

```diff
@@ -86,15 +83,17 @@ mod algorithm;
 mod block;
 mod error;
 mod instance;
-mod memory;
+mod memory_view;
 mod params;
+mod variable_hash;
 mod version;
 
 pub use crate::{
     algorithm::Algorithm,
     block::Block,
     error::{Error, Result},
-    params::{Params, ParamsBuilder},
+    instance::{Argon2, MAX_PWD_LEN, MAX_SALT_LEN, MAX_SECRET_LEN, MIN_SALT_LEN},
```

Comment:
If this is the new home of the `Argon2` type, it might make sense to rename the module, although I can't think of a better name than `argon2`.

Alternatively the `Argon2` type could live in the toplevel module of the crate.